src/config.ts

```diff
@@ -44,3 +44,5 @@ export const generateStorageKey = (
 ): string => `${topic}:${key}:${identifier}`;
 
 export const FEED_SURVEY_INTERVAL = 30;
+
+export const WEBAPP_MAGIC_IMAGE_PREFIX = `/imageGenerator`;
```

Comment:
maybe better if its `/image-generator`, don't like mixing casing in URLs?

Reply:
Good point. Updated it in https://github.com/dailydotdev/apps/pull/3766
```suggestion
export const WEBAPP_MAGIC_IMAGE_PREFIX = `/image-generator`;
```